core/web/middleware.go

```diff
@@ -60,23 +59,19 @@ func NewEmbedFileSystem(efs embed.FS, pathPrefix string) ServeFileSystem {
 }
 
 // Exists implements the ServeFileSystem interface.
-func (e *EmbedFileSystem) Exists(prefix string, filepath string) bool {
-	found := false
+func (e *EmbedFileSystem) Exists(prefix string, filepath string) (found bool, err error) {
 	if p := path.Base(strings.TrimPrefix(filepath, prefix)); len(p) < len(filepath) {
-		//nolint:errcheck
```

Comment:
This was only flagged for missing an explanation, but it was suppressing real errors too.